src/app/firebase-app.ts

```diff
@@ -215,6 +219,25 @@ export class FirebaseApp implements App {
     return this.ensureService_(name, () => init(this));
   }
 
+  /**
+   * Returns `true` if this app was initialized with auto-initialization.
+   * 
+   * @internal
+   */
+  public autoInit(): boolean {
+    return this.autoInit_;
+  }
+
+  /**
+   * Returns `true` if the FirebaseApp instance was initialized with a custom
+   * Credential.
```

Comment:
I'd lowercase "credential" unless it's a literal.

And I would backtick `FirebaseApp` as a literal.